src/coffee/contentscript.coffee

```diff
@@ -1,31 +1,50 @@
 excludeRooms = []
-
+unreadRoomsName = []
 timer = 0
-handleDOM = ->
-  return if timer
 
-  timer = setTimeout ->
-    $.each excludeRooms, (_, roomId) ->
-      selector = "#_roomListItems li[data-rid=#{roomId}]"
+getMentionCounts = ->
+  $('#_roomListArea .mention').length
 
+getUnreadRoomsName = ->
+  l_unreadRoomsName = $.map($('#_roomListItems .roomUnread .chatListTitleArea'), $.text)
+  if l_unreadRoomsName.length == 0
+    l_unreadRoomsName = [ 'No message!' ]
```

Comment:
@ducfilan 

I think that the following title is displayed in this code if we have no message.

```
[1](0) - No message!
```

I would like to change to the following title.

```
[0] - ChatWork
or
[0] - No message!
```

What do you think?